app_legacy/Helpers/database/game.php

```diff
@@ -12,7 +12,7 @@ function getGameData(int $gameID): ?array
         return null;
     }
 
-    $query = "SELECT gd.ID, gd.Title, gd.ConsoleID, gd.ForumTopicID, IFNULL( gd.Flags, 0 ) AS Flags, gd.ImageIcon, gd.ImageTitle, gd.ImageIngame, gd.ImageBoxArt, gd.Publisher, gd.Developer, gd.Genre, gd.Released, gd.IsFinal, c.Name AS ConsoleName, c.ID AS ConsoleID, gd.RichPresencePatch
+    $query = "SELECT gd.ID, gd.Title, gd.ConsoleID, gd.ForumTopicID, IFNULL( gd.Flags, 0 ) AS Flags, gd.ImageIcon, gd.ImageTitle, gd.ImageIngame, gd.ImageBoxArt, gd.Publisher, gd.Developer, gd.Genre, gd.Released, gd.IsFinal, gd.RAGuideURL, c.Name AS ConsoleName, c.ID AS ConsoleID, gd.RichPresencePatch
```

Comment:
This does cause `RAGuideURL` to be attached to `API_GetGameExtended.php` and `API_GetGameInfoAndUserProgress.php`. This is additive so it should not be a breaking change. The value will either be `null` or a string.